command/volume_snapshot_create.go

```diff
@@ -30,6 +31,17 @@ Usage: nomad volume snapshot create <volume id> [snapshot_name]
   When ACLs are enabled, this command requires a token with the
   'csi-write-volume' capability for the volume's namespace.
 
+
+Snapshot Create Options:
+
+  -secret
+    Secrets to pass to the plugin to create snapshot. Accepts multiple
+    flags in the form -secret key=value
+
+  -parameter
+    Parameters to pass to the plugin to create a snapshot. Accepts multiple
+    flags in the form -parameter key=value
+
```

Comment:
Command specific options usually go after `General Options`.

Reply:
Not on this PR, but it's also missing docs for the `verbose` flag?

Reply:
Good catch, will fix.